holmes/plugins/toolsets/azuremonitor_metrics/utils.py

```
@@ -0,0 +1,714 @@
+        if cluster_name not in enhanced_query:
+            logging.warning("No cluster filter was added to the query")
+        
+        logging.debug(f"Enhanced PromQL query: {promql_query} -> {enhanced_query}")
+        return enhanced_query
+        
+    except Exception as e:
+        logging.warning(f"Failed to enhance PromQL query with cluster filter: {e}")
+        return promql_query
+
+
+import os
```

Ruff (0.12.2)

Redefinition of unused `os` from line 6

Remove definition: `os`

(F811)